interface/forms/LBF/new.php

```diff
@@ -321,43 +321,24 @@ function end_row()
 ?>
 <html>
 <head>
-    <?php Header::setupHeader(['opener', 'common', 'datetime-picker', 'jquery-ui',]); ?>
+    <?php Header::setupHeader(['opener', 'common', 'datetime-picker']); ?>
 
     <style>
 
-        td, input, select, textarea {
-            font-family: Arial, Helvetica, sans-serif;
-            font-size: 10pt;
-        }
-
-        .table > tbody > tr > td {
-            border-top: 0;
-        }
-
```

Comment:
I removed the `.table > tbody > tr > td` style, then should I add `border-top-0` in each td?

Reply:
@stu01509 That's correct -- unless it doesn't look good when you load it up again